### PR TITLE
RTBA-52/khaliullin/registration/error

### PR DESCRIPTION
**Pull request by Rishat111**
Made tests for error.

### CODE REVIEW

**Comment by Rishat111** [comment]
https://jira.pasv.us/browse/RBTA-52

File: wdio.conf.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ exports.config = {
     //======================================
     './test/specs/reg/*.js',   //DO NOT REMOVE. This is short path to run all the specs.
     './test/specs/reg/**/*.js' //When you're writing new tests, just add you line between the lines above.
+
```
Comment (petrokolosok) [inline wdio.conf.js]: All tests passed.
Could you remove this empty line, please.

Comment (Rishat111) [reply wdio.conf.js]: ok, already did it!

**Comment by petrokolosok** [comment]
Great!

ср, 22 трав. 2019 о 22:31 Rishat Khaliullin <notifications@github.com> пише:

**Pull request merged**